src/DependencyInjection/Compiler/FormTwigTemplateCompilerPass.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
+
 /*
  * This file is part of the Symfony2 PhoneNumberBundle.
```

Comment:
What do you think of the file header update?

Reply:
This is a good idea!

I discovered recently that's is something supported by php-cs : https://github.com/symfony/symfony/blob/7.1/.php-cs-fixer.dist.php#L16

Maybe we can open an issue / PR in order to decide of a header we want to have in all files?